components/ws-manager-api/go/config/config.go

```diff
@@ -113,11 +109,13 @@ type Configuration struct {
 	RegistryFacadeHost string `json:"registryFacadeHost"`
 	// Cluster host under which workspaces are served, e.g. ws-eu11.gitpod.io
 	WorkspaceClusterHost string `json:"workspaceClusterHost"`
+	// WorkspaceClasses provide different resource classes for workspaces
+	WorkspaceClasses map[string]*WorkspaceClass `json:"workspaceClass"`
 }
 
-// AllContainerConfiguration contains the configuration for all container in a workspace pod
-type AllContainerConfiguration struct {
-	Workspace ContainerConfiguration `json:"workspace"`
+type WorkspaceClass struct {
+	Container ContainerConfiguration            `json:"container"`
+	Templates WorkspacePodTemplateConfiguration `json:"templates"`
```

Comment:
maybe singular `Template`?

Reply:
The `WorkspacePodTemplateConfiguration` contains multiple templates, hence the plural